test/unit/test-atomic.cpp

```diff
@@ -70,7 +70,7 @@ void testAtomicFunctionBasic(){
   dest[1] = (T)N;
 
   RAJA::forall<ExecPolicy>(seg,
-    [=] RAJA_HOST_DEVICE (RAJA::Index_type i){
+    [=] RAJA_HOST_DEVICE (RAJA::Index_type){
```

Comment:
Alternatively, there is a RAJA macro in defines.hpp that can be used to removed the named argument to prevent compiler warnings.  It makes it easier to find such things in the code if needed.